areas.wotw

```diff
@@ -2473,7 +2479,8 @@ anchor EastHollow.AfterBeetleFight at -28, -4233:  # At the breakable floor to B
       DoubleJump, TripleJump, Dash
       Bash  # Go to the right to make the mantis spawn if it's not here
     unsafe:
-      DoubleJump, SentryJump=1 OR Dash
+      DoubleJump, SentryJump=1 OR Dash  # @validator How does dash work here?
```

Comment:
Just for clarification: unsafe difficulty paths don't actually get validated, only all other difficulties receive validation